doc/source/whatsnew/v1.0.0.rst

```diff
@@ -206,6 +206,7 @@ I/O
 - Improve infinity parsing. :meth:`read_csv` now interprets ``Infinity``, ``+Infinity``, ``-Infinity`` as floating point values (:issue:`10065`)
 - Bug in :meth:`DataFrame.to_csv` where values were truncated when the length of ``na_rep`` was shorter than the text input data. (:issue:`25099`)
 - Bug in :func:`DataFrame.to_string` where values were truncated using display options instead of outputting the full content (:issue:`9784`)
+- Docs for the deprecated :func:`DataFrame.to_msgpack` give an example usage of the recommended pyarrow
```

Comment:
this note is not needed

Reply:
Got it. Removed.

Reply:
@jreback  -- Dont leave people hanging --  notes cost nothing.  Since the deprecation caused the problem to a good code. Hint to solution - instead of a blind door.  msgpack are very concise object. pyarrow  dont know.. leave breadcrumbs..